database/postgres/README.md

```diff
@@ -5,6 +5,7 @@
 | URL Query  | WithInstance Config | Description |
 |------------|---------------------|-------------|
 | `x-migrations-table` | `MigrationsTable` | Name of the migrations table |
+| `x-schema-name` | `SchemaName` | Schema name of the migratations table |
```

Comment:
1. Rename to `x-migrations-schema-name` if only used for the migrations table. Otherwise, if we use the name `x-schema-name`, we should set the current schema after connecting.
1. Document how this interacts with `search_path`
1. `migratations` -> `migrations`

Reply:
> Rename to `x-migrations-schema-name` if only used for the migrations table. Otherwise, if we use the name `x-schema-name`, we should set the current schema after connecting.

Done and pushed.

> migratations -> migrations

Done and pushd.

Reply:
I close this thread because I have pushed a second implementation.